### ISSUE TITLE
SSL certificate will be distrusted

### ISSUE THREAD

**Issue by saiichihashimoto** [Automattic/mongoose#6004]
![image](https://user-images.githubusercontent.com/2819256/35021996-641f9bc8-fae8-11e7-899c-1671f5e146e4.png)

Looks like the [documented method](http://mongoosejs.com/docs/browser.html) of including mongoose in the browser will be distrusted soon. What's the intended alternative?

**Comment by vkarpov15** [comment]
We [removed support for the pre-compiled browser bundle](https://github.com/Automattic/mongoose/blob/master/History.md#500-rc0--2017-12-28) in 5.0.0-rc0, see #5895 and will be removing this from the new docs site in #6007. We'll update the 4.x docs to point to a new cloudfront link but we're not going to support this for new releases.

**Comment by saiichihashimoto** [comment]
That makes sense! I'd think that, until the [issues with webpack](https://github.com/Automattic/mongoose/issues/5993#issuecomment-358418353) are solved, having a working pre-compiled browser bundle is still essential. But this warning isn't breaking so... keep up the good work!!!

**Comment by vkarpov15** [comment]
Thanks! Yeah we definitely need to do some work to get mongoose to behave properly with webpack. We started shipping the browser bundle in early 2015, long before webpack became this popular, and the whole browser bundle idea hasn't aged very gracefully. Going forward I'm thinking we should focus more on making mongoose easier to build with webpack, browserify, and babel rather than maintaining the pre-built bundle.

**Comment by saiichihashimoto** [comment]
Sounds awesome! Since the browser bundle is going to be removed, this seems like a wont-fix. Feel free to close!

**Comment by vkarpov15** [comment]
I'm going to close this one for similar reasons as https://github.com/apex/up/issues/528 , upon further inspection it looks like its a cloudfront-wide issue, not just an issue with our cloudfront setup. Not much we can do other than find some hosting service that doesn't use symantec for HTTPS,

As an aside, I think we found a solution to the webpack issue, see #5842 and https://github.com/Automattic/mongoose/commit/31b84f042d5786cbb674daa9974155f5af9e8b62. We also removed the offending `browser.html` page that was recommending that cloudfront link.
